include/fastq_conversion.hpp

```diff
@@ -0,0 +1,14 @@
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/io/sequence_file/all.hpp>
+#include <seqan3/io/sequence_file/input.hpp>
+#include <seqan3/io/sequence_file/output.hpp>
```

Comment:
`#include <seqan3/io/sequence_file/all.hpp>` should already contain input and output... Please remove the two include statements.